src/test/java/com/esendex/java/sdk/BaseTest.java

```diff
@@ -1,57 +1,9 @@
 package com.esendex.java.sdk;
 
-import esendex.sdk.java.EsendexException;
 import esendex.sdk.java.EsendexProperties;
-import esendex.sdk.java.ServiceFactory;
-import esendex.sdk.java.service.auth.UserPassword;
-import esendex.sdk.java.service.impl.IServiceFactory;
 
 public abstract class BaseTest {
-
-    public static boolean IsSessionMode = false;
-
-    public static final String USER;
-    public static final String PASSWORD;
-    public static final String ACCOUNT;
-    public static final String DESTINATION_NUMBER;
-	private static final UserPassword userPassword;
-
-
     static {
-        EsendexTestProperties testProperties = EsendexTestProperties.instance();
-
-        USER = testProperties.getUsername();
-        PASSWORD = testProperties.getPassword();
-        ACCOUNT = testProperties.getAccount();
-        DESTINATION_NUMBER = testProperties.getDestinationNumber();
-
-        userPassword = new UserPassword(USER, PASSWORD);
-
-        String domain = testProperties.getDomain();
-
-        if(domain != null)
-            EsendexProperties.instance().setProperty("esendex.domain", domain);
-    }
-
-    public static IServiceFactory getFactory() throws EsendexException {
-        return IsSessionMode
-                ? getBasicFactory()
-                : getSessionFactory();
-    }
-
-    private static IServiceFactory getBasicFactory() {
-
-        if (basicFactory == null)
-            basicFactory = ServiceFactory.createBasicAuthenticatingFactory(userPassword);
-        return basicFactory;
-    }
-    private static IServiceFactory basicFactory;
-
-    private static IServiceFactory getSessionFactory() throws EsendexException {
-
-        if (sessionFactory == null)
-            sessionFactory = ServiceFactory.createSessionAuthenticatingFactory(userPassword);
-        return sessionFactory;
+        EsendexProperties.instance().setProperty("esendex.secure", "false");

```

Comment:
Half of me want's to suggest putting this on the 'Before' of the tests as it seems somewhat pointless to have this Base class which the others extend now.